e2e-polybft/property/property_test.go

```diff
@@ -55,3 +56,74 @@ func TestProperty_DifferentVotingPower(t *testing.T) {
 		require.NoError(t, cluster.WaitForBlock(numBlocks, blockTime*time.Duration(numBlocks)))
 	})
 }
+
+func TestProperty_DropValidators(t *testing.T) {
+	t.Parallel()
+
+	const (
+		blockTime = time.Second * 4
+	)
+
+	rapid.Check(t, func(tt *rapid.T) {
+		var (
+			numNodes  = rapid.Uint64Range(5, 8).Draw(tt, "number of cluster nodes")
+			epochSize = rapid.OneOf(rapid.Just(4), rapid.Just(10)).Draw(tt, "epoch size")
+		)
+
+		cluster := framework.NewPropertyTestCluster(t, int(numNodes),
+			framework.WithEpochSize(epochSize),
+			framework.WithSecretsCallback(func(adresses []types.Address, config *framework.TestClusterConfig) {
+				for range adresses {
+					config.StakeAmounts = append(config.StakeAmounts, big.NewInt(20))
+				}
+			}))
+		defer cluster.Stop()
+
+		t.Logf("Test %v, run with %d nodes, epoch size: %d",
+			filepath.Base(cluster.Config.LogsDir), numNodes, epochSize)
+
+		cluster.WaitForReady(t)
+
+		// stop first validator, block production should continue
+		cluster.Servers[0].Stop()
+		activeValidator := cluster.Servers[numNodes-1]
+		currentBlock, err := activeValidator.JSONRPC().Eth().GetBlockByNumber(ethgo.Latest, false)
+		require.NoError(t, err)
```

Comment:
Use `eth_blockNumber` function (accomodate usage of `currentBlock` ref), since you are interested only in block number?

```suggestion
		currentBlock, err := activeValidator.JSONRPC().Eth().BlockNumber()
```

Reply:
fix d8645f3